.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: java
+sudo: false
+script:
+  - mvn clean package assembly:single
```

Comment:
What is your opinion on added `tidy:check license:check spotbugs:check` guards here?

Reply:
I knew only `tidy`, added that together with `spotbugs` in https://github.com/FAIRDataTeam/OpenRefine-metadata-extension/pull/1/commits/8c5871ba9e3ed2c2e3324327e6da3560dd81c5c3 but by `license` you mean this one https://www.mojohaus.org/license-maven-plugin/ ?

Reply:
Added license plugin to add license to .java files.

Reply:
License plugin added in https://github.com/FAIRDataTeam/OpenRefine-metadata-extension/pull/1/commits/7eccc6b166dcbcc82d30b3d78184c257e110952a